src/modules/utils/panel/screens/3dprinter/WatchScreen.cpp

```diff
@@ -319,3 +293,109 @@ const char *WatchScreen::get_network()
 
     return NULL;
 }
+
+void WatchScreen::refresh_screen(bool clear)
```

Comment:
IMO this does not belong in Watchscreen at all, and should just be handled in on_refresh()